apps/emqx_management/src/emqx_mgmt_data_backup.erl

```diff
@@ -454,6 +456,17 @@ import_acl_mnesia(Acls, FromVersion) when FromVersion =:= "4.0" orelse
 
 import_acl_mnesia(Acls, _) ->
     do_import_acl_mnesia(Acls).
+
+do_import_auth_mnesia_4_2(Auths) ->
+    case ets:info(emqx_user) of
+        undefined -> ok;
+        _ ->
+            CreatedAt = erlang:system_time(millisecond),
+            lists:foreach(fun(#{<<"login">> := Login,
+                                <<"password">> := Password}) ->
+                            mnesia:dirty_write({emqx_user, {username, Login}, Password, CreatedAt})
```

Comment:
Actually, there is a design compatibility problem herel. Here we have hard-coded the type `username`. But this type depends on the `auth.mnesia.as` value in the `emqx_auth_mnesia.conf` in version 4.1/4.2.

Unfortunately, the data files generated from older versions do not carry this type...

Reply:
We need to find a way to maintain compatibility. We discussed this before by adding command-line arguments.

For example :
```
emqx_ctl import data import v4.2.json --env '{"auth.mnesia.as": "username"}'
```

Do you have any better suggestions?